.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!-- Thanks for submitting a pull request for Destination Sol! :-)
+Please fill in some brief details below about the PR.
+If it contains source code please make sure to do everything in the pre pull request checklist first.
+If you add unit tests we'll love you forever! -->
+
+# Description
+> Please describe what this PR does and why it does it
+
+# Testing
+> If the PR is testable, please describe how to test it here
+
+# Outstanding Work
+- [ ] If there is work still outstanding, please include it here
+- [ ] if there is nothing outstanding, remove this section.
+
+# Notes
+> Include any extra notes here
+
+### Pre Pull Request Checklist:
+<!-- When scanning the code with SonarLint, we don't except you to fix every single thing, just fix yours, and a few extra-->
```

Comment:
"just don't introduce any new issues, and maybe even fix a few existing ones" sounds better IMO.